nfs/file.go

```diff
@@ -177,8 +178,36 @@ func (f *File) Close() error {
 	return nil
 }
 
+// Seek sets the offset for the next Read or Write to offset, interpreted according to whence.
+// This method implements Seeker interface.
+func (f *File) Seek(offset int64, whence int) (int64, error) {
+
+	// It would be nice to try to validate the offset here.
+	// However, as we're working with the shared file system, the file
+	// size might even change between NFSPROC3_GETATTR call and
+	// Seek() call, so don't even try to validate it.
+	// The only disadvantage of not knowing the current file size is that
+	// we cannot do io.SeekEnd seeks.
+	switch whence {
+	case io.SeekStart:
+		if offset < 0 {
+			return int64(f.curr), errors.New("offset cannot be negative")
+		}
+		f.curr = uint64(offset)
+		return int64(f.curr), nil
+	case io.SeekCurrent:
+		f.curr = uint64(int64(f.curr) + offset)
+		return int64(f.curr), nil
+	case io.SeekEnd:
+		return int64(f.curr), errors.New("SeekEnd is not supported yet")
+	default:
+		// This indicates serious programming error
+		panic("Invalid whence")
```

Comment:
I am thinking that panic might be heavy handed here? My main concern is people using this as a client may be surprised if it causes an unhandled panic unexpectedly.

Reply:
I cannot imagine situation other than developer's mistake that could lead to "whence" values being something else than those three values (since this is more something you get from outside). But no problem, I'll update the code to just return an error instead.

Reply:
Sorry, I meant "not something" of course...

Reply:
Done!